msal_extensions/persistence.py

```diff
@@ -150,8 +153,13 @@ def save(self, content):
 
     def load(self):
         with self._Keychain() as locker:
-            return locker.get_generic_password(
-                self._service_name, self._account_name)
+            from .osx import KeychainError
+            try:
+                return locker.get_generic_password(
+                    self._service_name, self._account_name)
+            except KeychainError as kce:
+                if kce.exit_status != KeychainError.ITEM_NOT_FOUND:
+                    raise
```

Comment:
Given that we now head to the direction of using `None` as an indication of ITEM_NOT_FOUND, we better make such intention explicit.

```suggestion
                if kce.exit_status == KeychainError.ITEM_NOT_FOUND:
                    return None  # Map ITEM_NOT_FOUND exception to None return value
                raise
```